sdk/fido/fido_util.cpp

```diff
@@ -189,6 +189,10 @@ std::string to_zone_net_node(const wwiv::sdk::fido::FidoAddress& a) {
   return StrCat(a.zone(), ":", to_net_node(a));
 }
 
+std::string to_zone_net_node_point(const wwiv::sdk::fido::FidoAddress& a) {
```

Comment:
Seems like I should just call that to_string and always include the point - @apamment Did you notice a place where we don't want the point included (I know little of FTN)